### DISCUSSION TITLE
关于中文与中文之间空格的排版问题

### DISCUSSION THREAD

**Discussion by fwqaaq** [reactjs/zh-hans.react.dev#1109] [General]
在 React 中，链接、加租、斜体的渲染幅度足够清晰

![image](https://user-images.githubusercontent.com/82551626/228470768-b9a968d2-299b-4ac0-be20-8ffc23018ab6.png)

![image](https://user-images.githubusercontent.com/82551626/228471007-f47cc2e6-2f44-4cb3-8482-36364fcf7e6f.png)

对于中文与中文之间的空格在这里是可以省略的，多处的空格会造成排版的不美观以及会有标点才会有的停顿感，对于粗体、斜体只要字样起到关注点就好了，没必要增加空格。

**Comment by awxiaoxian2020** [comment]
from @QC-L：
这个是要看字体，如果按中文排版来说，是不需要加的。当时是官方的英文字体加粗也没有辨识度，所以加了空格。

**Comment by awxiaoxian2020** [comment]
其实我们如果现在进行更改，还需要考虑一个合适的正则表达式处理之前的文档。

**Reply by fwqaaq** [reply]
可以使用 markdownlint2 进行匹配，但是可能会有遗漏

**Comment by QC-L** [comment]
感谢建议，但我觉得没必要修改。这里是为了突出链接本身，增加辨识度。

**Reply by fwqaaq** [reply]
可是 React 这辨识度，怕是色盲都知道它是链接🤔

**Comment by awxiaoxian2020** [comment]
[中文文案排版指南](https://github.com/sparanoid/chinese-copywriting-guidelines/blob/master/README.zh-Hans.md) 对于此问题的表述是“争议”，没有更多的人对这个议题 vote up，我建议暂时保持现状。

**Reply by fwqaaq** [reply]
有道理，我大概搜集了一些它列出的网站

> https://www.apple.com.cn/
> ![image](https://user-images.githubusercontent.com/82551626/228487967-c1f63728-02fa-4d71-9c74-fe941eab1ff2.png)
>
> https://visualstudio.microsoft.com/zh-hans/
> ![image](https://user-images.githubusercontent.com/82551626/228489182-1a181837-33d7-429b-a1a2-de4ec915a2c7.png)
>
> https://leancloud.cn/docs/index.html
> ![image](https://user-images.githubusercontent.com/82551626/228490093-72e12802-069e-4cdd-872a-d49ead11efab.png)
>
>https://aws.amazon.com/cn/blogs/china/aws-chatbot-now-integrates-with-microsoft-teams/
>![image](https://user-images.githubusercontent.com/82551626/228491300-9127ca3f-127c-48eb-82f5-19f628b57604.png)
>
>https://web.dev/vitals/
>![image](https://user-images.githubusercontent.com/82551626/228492521-8a2d924e-80e2-4f55-84ac-bce1bc3c8e52.png)

**Comment by awxiaoxian2020** [comment]
就我个人而言，我觉得斜体间不加空格显得拥挤和奇怪，我一直对是否保留斜体有不同态度。

既然斜体，英文等前后都需要添加空格，也将粗体、链接单独拿出来，作为强调，从这个角度上说，加空格不是不可以接受的。

其实最主要的问题还是对先前的文档调整的问题，因为 React 有自己的 lint，拿 Markdownlint 调整后可能会出一些问题。暂时保持一致吧，我已更新了 wiki 相关内容。

**Comment by awxiaoxian2020** [comment]
https://github.com/reactjs/react.dev/issues/5443

当时可能考虑到的是这个 issue

**Comment by earthaYan** [comment]
可以考虑搞一个格式检查插件，比如空格，标点符号之类的检查，人工检查工作量非常大且容易遗漏

**Comment by Yucohny** [comment]
目前仍然采取当出现加粗、链接等情况时中文与中文之间需要使用空格的方式。
